doc/doxygen/header.html

```diff
@@ -30,6 +30,15 @@
       <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+        <script>
+            function displaySR(){
+            var rhtml=(document.getElementById('MSearchResults').contentWindow.document.body.innerHTML).replace(/href=\"\.\.\//g,'href="');
```

Comment:
This results in a Content Security Policy violation on Firefox when accessing a locally generated documentation.

The origin of this script is `file:///`, the search results are an `iframe` with `javascript:void(0)` as origin. Apparently, this is considered cross origin.

Reply:
Note: Just moving the body of the function directly into the `onsubmit=""` doesn't help. (Which makes sense, it still has origin `file:///`.)

Reply:
OK, for me this seems like a fix is not worth it

Reply:
i like the method of extracting the result from the Iframe into a div